recipe/meta.yaml

```diff
@@ -11,21 +11,20 @@ source:
 
 build:
     number: 0
-    features:
-        - vc9  # [win and py27]
-        - vc10  # [win and py34]
-        - vc14  # [win and py>=35]
-    skip: true  # [win and py>35]
+    run_exports:
+        - {{ pin_subpackage('zlib', max_pin='x.x.x') }}
 
 requirements:
     build:
-        - python  # [win]
-        - cmake  # [win]
-        - msinttypes  # [win and py<35]
+        - cmake       # [win]
+        - msinttypes  # [win and vc<14]
```

Comment:
I wonder if this is still needed with the new compilers.

Reply:
Yes, we don't automatically add this in the compiler packages.  The change here is to associate it with a specific vc version, rather than a python version (more direct association)

Reply:
In windows, new compilers are just an activation script for MSVC

Reply:
Agree it might be a good idea to make this a dependency of the compiler package for VS 2008 if possible. Not having it is a common problem.

Reply:
not everything using VS 2008 needs msinttypes.

Reply:
> not everything using VS 2008 needs msinttypes.

Yep. It is probably better to leave this unchanged.

Reply:
Not following how it hurts to always include it. Just smooths over the brokenness of VS 2008.

Reply:
- not worth the effort for the little advantage;
- will add extra bytes of unnecessary downloads;
- some packages work around this issue making this package optional, its presence should always be explicit in those case.

Reply:
I think the more important thing is avoiding complications with packages that implement this functionality themselves.  I have seen a few recipes break when they have their own internal workaround, and this header is also present.

Other than that, I'm not opposed to always including it.  It's tiny, and the effort required is small (just add a dep to the VS2008 recipe).